pulsar-websocket/src/main/java/com/yahoo/pulsar/websocket/ConsumerHandler.java

```diff
@@ -139,7 +139,11 @@ public void onWebSocketText(String message) {
             return;
         }
 
-        consumer.acknowledgeAsync(msgId);
+        consumer.acknowledgeAsync(msgId).exceptionally(t -> {
```

Comment:
I don't think it's very useful since we don't know what to do if we fail to acknowledge..
The reason is that acknowledging a message operation doesn't have an ack, so we just wait to write on the socket. If that fails, the broker is anyway going to send the message again, at which point another ack will be issued.